charts/hivemq-platform/tests/services/hivemq_rest_api_services_test.yaml

```diff
@@ -134,7 +134,7 @@ tests:
           hivemqProxyProtocol: true
     asserts:
       - failedTemplate:
-          errorMessage: Service type rest-api with container port 8888 is using PROXY protocol value. PROXY protocol is only supported for MQTT and WebSocket services
+          errorPattern: Service type `rest-api` with container port `8888` is using `hivemqProxyProtocol` value. HiveMQ PROXY protocol is only supported by MQTT and WebSocket services
```

Comment:
Container port?

Reply:
Same comment is applicable here: https://github.com/hivemq/helm-charts/pull/403#discussion_r1866225099